src/test/scala/cromwell/CromwellTestkitSpec.scala

```diff
@@ -203,6 +203,27 @@ with DefaultTimeout with ImplicitSender with WordSpecLike with Matchers with Bef
     (wma, workflowId)
   }
 
+  def runWdlAndAssertOutputs2(sampleWdl: SampleWdl,
```

Comment:
This looks suspiciously like something I was futzing with months ago but I don't see how it'd end up here